src/statsmodels/formula.jl

```diff
@@ -40,6 +40,9 @@ type Terms
     intercept::Bool       # is there an intercept column in the model matrix?
 end
 
+import Base.==
+==(t1::Terms, t2::Terms) = all( getfield(t1, f)==getfield(t2, ) for f in fieldnames(t1))
```

Comment:
I just meant that you should write `Base.:(==)(...) = ...`. Though I just noticed `f` seems to be missing in `getfield(t2, )`?